nVolve/Features/Filters/FilterViewModel.swift

```diff
@@ -67,4 +84,45 @@ class FilterViewModel: ObservableObject {
             return matches
         }
     }
+
+    
+    func areStringsSimilar(_ string1: String, _ string2: String, threshold: Int = 3) -> Bool {
```

Comment:
leetcode lookin ass function

Reply:
lol This just checks if two strings are similar. The reason for this is if we have a filter for "spiritual" theme and the theme pulled from api says "spirituality", they are similar enough so it is a match.